### PR TITLE
pythonPackages.yowsup: init at v2.5.2

### PR DESCRIPTION
**Pull request by Ma27**
###### Motivation for this change

The `yowsup` library is needed for the WhatsApp script provided by weechat: https://github.com/weechat/scripts/blob/master/python/whatsapp.py

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/top-level/python-packages.nix [hunk 1]
```diff
@@ -26537,6 +26537,30 @@ EOF
...
 
+  yowsup = buildPythonPackage rec {
+    name = "${pname}-${version}";
+    pname = "yowsup";
+    version = "v2.5.2";
+
+    src = pkgs.fetchFromGitHub {
+      owner = "tgalal";
+      repo = "yowsup";
+      rev = "v2.5.2";
```
Comment (vyp) [inline pkgs/top-level/python-packages.nix]: Can just be `rev = version;`.

File: pkgs/top-level/python-packages.nix [hunk 2]
```diff
@@ -26537,6 +26537,30 @@ EOF
     };
   };
 
+  yowsup = buildPythonPackage rec {
```
Comment (Mic92) [inline pkgs/top-level/python-packages.nix]: Please read the header of this file. We want to move new or updated expressions to `pkgs/development/python-modules/<name>/default.nix`

**Comment by Ma27** [comment]
@Mic92 done :-)

**Comment by Mic92** [comment]
Let me now, whether the weechat plugin is working well.

**Pull request merged**